internal/vds/axis.h

```diff
@@ -0,0 +1,38 @@
+#ifndef AXIS_H
+#define AXIS_H
+
+#include <memory>
+#include <string>
+
+#include <OpenVDS/OpenVDS.h>
+
+#include "vds.h"
```

Comment:
If we are happy with adding more files, I think splitting the header up would be a good idea. Maybe something like `vdsslice_definitions.h(pp)` could make sense for the numerations etc?

I would say that the entry point, at least the header, could also be generic if one renames some function parameters, (e.g. parameters `vds` -> `url` or similar). This means we could create something like `seismic_entry_point.hpp` or `poststack_entry_point.hpp`. The whole VDS specific part in the implemention of the interface comes in the implementation `vds.cpp`. This could then be something like `vds_poststack_entry_point.cpp`.

Reply:
If we want to be very fine-grained we might also want to have `poststack_entrypoint_helper.cpp` which implements `response_delete` and potentially `json_axis` as these functions are not VDS specific.